config/config_test.go

```diff
@@ -87,95 +86,14 @@ func makeCfgService() config.Service {
 	return cfgService
 }
 
-func TestNewAttrConf(t *testing.T) {
```

Comment:
Why are we removing these tests instead of converting to tests of `Validate`?

Reply:
Done: https://github.com/viamrobotics/slam/pull/185/files#diff-373f5578b8f362a364b50b0d01919a57da64c128547ca10d49df8ae422237c82R18

Reply:
I also added tests for the actual messages of the errors